docs/_documentations/mdt-che-overview.md

```diff
@@ -21,3 +21,24 @@ The Eclipse tools are [open source](https://github.com/eclipse/codewind-eclipse)
 To get started, see [Getting started with Codewind](mdteclipsegettingstarted.html).
 
 To uninstall Codewind, see [Uninstalling Codewind for Eclipse](mdteclipseuninstall.html)
+
+## Codewind ODO extension 
+Extension to Codewind providing support for ODO projects: https://codewind.dev
+
+### Add additional rules to support Codewind ODO extension
+The ODO extension needs to add additional rules for accessing OpenShift resources:
+1. In your home directory, run the following command to clone the ODO extension repository:
+`git clone https://github.com/eclipse/codewind-odo-extension`
+2. Login to your OpenShift/OKD cluster
+3. Go into `~/codewind-odo-extension/odo-RBAC` then run the following commands to add additional rules:
+`kubectl apply -f codewind-odoclusterrole.yaml`
+`kubectl apply -f codewind-odoclusterrolebinding.yaml`
+
+### Install Codewind ODO extension manually
```

Comment:
@GeekArthur Is there any reason why we need to include manually install odo extension info in the doc? Don't we always install it automatically for user?

Reply:
@elsony It’s only for installer doesn’t work properly, in order to not block user use the ODO extension we provide manual install

Reply:
@elsony But for landing page, I think we can move the manual install to trouble shooting page for user hit the auto install doesn't work properly issue. And don't include manual install in our main Codewind for Eclipse Che Overview Section